.travis.yml

```diff
@@ -28,6 +24,8 @@ branches:
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/ # tags
 
 jobs:
+  allow_failures:
+    - julia: nightly
```

Comment:
In line 34, please add `AND fork = false`. I'm not sure why my tests did not run for this, but yours do.

Reply:
Great, It works now.